notebooks/VectorSearch_QuestionRetrieval.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p><code>RAFT</code> is an acronym- for consistency can we use RAFT instead <code>Raft</code> please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>